src/typescript/frontend/public/locales/en-US.json

```diff
@@ -61,5 +61,7 @@
   "Cost to deploy": "Cost to deploy",
   "Your balance": "Your balance",
   "Your transaction may fail due to gas costs.": "Your transaction may fail due to gas costs.",
-  "Balance": "Balance"
+  "Balance": "Balance",
```

Comment:
- [ ] Add `maintenance` and whatever else happens once you try to build it
- [ ] Mark this comment resolved once done